airflow/provider_info.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "package-name": {
+      "description": "Package name available under which the package is available in the PyPI repository.",
+      "type": "string"
+    },
+    "name": {
+      "description": "Provider name",
+      "type": "string"
+    },
+    "description": {
+      "description": "Information about the package in RST format",
+      "type": "string"
+    },
+    "versions": {
+      "description": "List of available versions in Pypi. Sorted descending according to release date.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
```

Comment:
I thought this wasn't part of get_provider_info?

Reply:
Good point. It does not have to be. I will remove!